map/map_height_fitter/schema/map_height_fitter.schema.json

```diff
@@ -10,6 +10,11 @@
           "type": "string",
           "description": "Node name of the map loader from which this map_height_fitter will retrieve its parameters",
           "default": "/map/pointcloud_map_loader"
+        },
+        "fit_target": {
+          "type": "string",
+          "description": "Node name of the map loader from which this map_height_fitter will retrieve its parameters",
```

Comment:
Hi. This description is incorrect.

Reply:
Ouch! Fixed in https://github.com/autowarefoundation/autoware.universe/pull/6340/commits/e4f1d9309541054f7c85753d2339444114d82630